src/lib/logMsg/logMsg.cpp

```diff
@@ -343,6 +343,9 @@ int             inSigHandler      = 0;
 char*           progName;                   /* needed for messages (and by lmLib) */
 char            progNameV[512];             /* where to store progName            */
 __thread char   transactionId[64] = "N/A";
+__thread char   srv[50 + 1]       = "N/A";     // FIXME: Maybe we should use limits.h ?
+__thread char   subsrv[51*10 +1]  = "N/A";     // FIXME: Maybe we should use limits.h ?
+__thread char   from[4*4 + 1]     = "N/A";     // Based in XXX.XXX.XXX.XXX
 
```

Comment:
srv ... I would prefer to write out the name "service", not especially long.
And, these names are very generic and need either to be static or have a prefix.
I guess static would be enough.

About 'from', it doesn't say much. Perhaps peerIp or clientIp would be more descriptive ?

About the default value, I'd leave it as is.
I really don't want any dependencies of common-lib in logMsg (which is a lower level library).

Now, about from[4*4 + 1]: just put 16, 4*4+1 is not even correct ...
Or, add a definition of NUMERICAL_IP_MAX_SIZE as 15 and then use NUMERICAL_IP_MAX_SIZE + 1.